packages/arb-token-bridge-ui/src/components/TransactionHistory/FailedTransactionsWarning.tsx

```diff
@@ -36,7 +36,7 @@ export const FailedTransactionsWarning = ({
   if (numFailedTransactions < 3) return null
 
   return (
-    <div className="flex items-center gap-1 rounded-md bg-orange p-2 text-base text-dark lg:flex-nowrap">
+    <div className="flex items-center gap-1 rounded-md bg-orange p-2 text-base text-dark lg:flex-nowrap mx-2">
```

Comment:
This margin was not intended for transaction history panel. Pls remove it.

<img width="880" alt="image" src="https://user-images.githubusercontent.com/7558499/229607264-62cd1bd9-ca00-47b2-a03b-b0005e61e7e3.png">